### PR TITLE
docs: A few cosmetic improvements in GKE guide

### PR DESCRIPTION
**Pull request by errordeveloper**
This change is [Reviewable](https://reviewable.io/reviews/cilium/cilium/10550)

### CODE REVIEW

**Review by tgraf — APPROVED** [review]

**Comment by tgraf** [comment]
test-docs-please

**Comment by tgraf** [comment]
```
00:01:42.908  Please fix the following documentation warnings:
00:01:42.908  /srv/Documentation/gettingstarted/k8s-install-gke.rst:58: WARNING: 'any' reference target not found: gcloud config get-value core/account
00:01:42.908  /srv/Documentation/gettingstarted/k8s-install-gke.rst:58: WARNING: 'any' reference target not found: GKE RBAC documentation
00:01:42.908  /srv/Documentation/gettingstarted/k8s-install-gke.rst:65: WARNING: 'any' reference target not found: GKE RBAC documentation
```

**Review by pchaigno — APPROVED** [review]

**Comment by errordeveloper** [comment]
test-docs-please

File: Documentation/gettingstarted/k8s-install-gke.rst [hunk 1]
```diff
@@ -34,14 +34,14 @@ Create a GKE Cluster
 
 You can apply any method to create a GKE cluster. The example given here is
 using the `Google Cloud SDK <https://cloud.google.com/sdk/>`_. This guide
-will create a cluster on zone ``europe-west4-a``; feel free to change the zone
-if you are in a different region of the globe.
+will create a single-zone cluster in ``europe-west4-a``; feel free to change
+the zone that is closer to where you are, you can create a regional cluster
```
Comment (tklauser) [inline Documentation/gettingstarted/k8s-install-gke.rst]: Nit:

```suggestion
the zone to one that is closer to where you are. You can create a regional cluster
```

Comment (errordeveloper) [reply Documentation/gettingstarted/k8s-install-gke.rst]: Thanks!

**Comment by errordeveloper** [comment]
test-docs-please

File: Documentation/gettingstarted/k8s-install-gke.rst (hunk 1 above)
Comment (tklauser) [reply Documentation/gettingstarted/k8s-install-gke.rst]: Still looks like the original in the latest push (9910c88), maybe you forgot to amend?

Comment (errordeveloper) [reply Documentation/gettingstarted/k8s-install-gke.rst]: Yeah, thanks again!

**Review by tklauser — APPROVED** [review]

**Comment by tklauser** [comment]
test-docs-please

**Pull request merged**